### PR TITLE
[11.x] Add support for AVIF image format in validation

### PR DESCRIPTION
**Pull request by robertotcestari**
Hello All! 

This PR adds support for `avif` image extensions in the Validation for image files. Avif [is now a Baseline 2024](https://caniuse.com/avif) format (it works across all browsers) and is now the recommended image format for the web.

It also adds tests and removes some duplicated assertions that i found inside the tests. 

Thanks!

### CODE REVIEW

File: tests/Validation/ValidationValidatorTest.php [hunk 1]
```diff
@@ -4846,12 +4846,6 @@ public function testValidateImage()
         $v = new Validator($trans, ['x' => $file2], ['x' => 'image']);
         $this->assertTrue($v->passes());
 
-        $file2 = $this->getMockBuilder(UploadedFile::class)->onlyMethods(['guessExtension', 'getClientOriginalExtension'])->setConstructorArgs($uploadedFile)->getMock();
-        $file2->expects($this->any())->method('guessExtension')->willReturn('jpg');
-        $file2->expects($this->any())->method('getClientOriginalExtension')->willReturn('jpg');
-        $v = new Validator($trans, ['x' => $file2], ['x' => 'image']);
-        $this->assertTrue($v->passes());
-
```
Comment (crynobone) [inline tests/Validation/ValidationValidatorTest.php:4854]: Does see why this need to be removed.

Comment (robertotcestari) [reply tests/Validation/ValidationValidatorTest.php:4854]: This assertion is duplicated with the above - lines 4843-4847

Comment (crynobone) [reply tests/Validation/ValidationValidatorTest.php:4854]: Might be best to not remove this on a PR that add additional feature. We can always submit test improvements separately.

Comment (robertotcestari) [reply tests/Validation/ValidationValidatorTest.php:4854]: ok, will fix this in a minute and add another PR!

Comment (robertotcestari) [reply tests/Validation/ValidationValidatorTest.php]: @crynobone done. Let me know if now this is ok!

**Comment by hafezdivandari** [comment]
https://github.com/laravel/framework/pull/36803#issuecomment-809621851

target 12.x maybe?

**Comment by robertotcestari** [comment]
> [#36803 (comment)](https://github.com/laravel/framework/pull/36803#issuecomment-809621851)
> 
> target 12.x maybe?

Supporting the AVIF format shouldn’t be considered a breaking change; in fact, the validator’s lack of support for it is more akin to a bug.

When you request image validation, you expect the validator to handle all popular formats. If it rejects AVIF, that’s unexpected behavior, rendering the Image File Validator useless.

If an application chooses not to support AVIF (or any other specific image format), it should explicitly define its accepted MIME types

**Comment by taylorotwell** [comment]
It is a breaking change as noted. People currently expect that rule to do what it is documented as doing.

**Comment by robertotcestari** [comment]
can i PR to 12.x then?

**Pull request closed without merging**